packages/hoprd/README.md

```diff
@@ -2,6 +2,13 @@
 
 Runs a HOPR Node and the HOPR Admin interface.
 
+When the Rest API is enabled, the node serves a Swagger UI to inspect and test
+the Rest API v2 at:
+
+http://localhost:3001/api/v2/\_swagger
```

Comment:
this URL didn't work for me, this did: `http://localhost:3001/api/v2/_swagger/`

Reply:
Typo, thanks.